src/client/dss.py

```diff
@@ -295,7 +479,7 @@ def make_dss_request():
 
 def download_file(filname, url, headers):
     with open(filname, 'wb') as handle:
-        resp = requests.get(url, headers=headers, stream=True)
+        resp = requests.get(url, headers=headers, stream=True, verify=common.global_vars['is_secure'])
```

Comment:
common keeps is_secure false. Pass nothing and it assumes true and validates certificates.crt